wolox-books/src/app/screens/shopping-cart/components/list/list.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Observable } from 'rxjs';
+import { Component } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { AppState } from 'src/app/app.state';
+import * as ShoppingCartActions from '../../../../store/shopping-cart.action';
+import { Book } from 'src/app/screens/book/book';
+
+@Component({
+  selector: 'shopping-cart-list',
+  templateUrl: './list.component.html',
+  styleUrls: ['./list.component.scss']
+})
+export class ListComponent {
+  books: Book[];
+
+  _open: boolean;
+
+  constructor(private store: Store<AppState>) {
+    this.store.select('shoppingCart').subscribe(element => {
+      this._open = element['modal'];
```

Comment:
is necessary `_open`? a glance in the template tells me that `this.open` should be the same. If you want to show me the converse, cool.

Reply:
Es necesario por que `_open` guarda el estado que se devuelve en `get open`, lo hago asi por que aunque este utilizando el estado de ngrx, para devolver el estado necesito un boolean no un Observable que es lo que se retornaria de llamar el store. https://github.com/wolox-training/jd-angular/pull/19#discussion_r441200187

Reply:
Ohh, okay. now i catch it